tests/integration/utils.sh

```diff
@@ -157,6 +158,123 @@ function teardown_single_container()
   fi
 }
 
+# Destroy all containers used in the tests
+teardown_containers()
+{
+  local distro
+  local i=0                # current step of tear down
+  local n="${#DISTROS[@]}" # total number of steps of tear down
+
+  for distro in "${DISTROS[@]}"; do
+    # progress message
+    i=$((i + 1))
+    say "[${i}/${n}] Removing ${distro} container."
+
+    teardown_single_container "kw-${distro}"
+  done
+}
+
+# Completely remove the container environment: the containers and the images
+function teardown_container_environment()
+{
+  local flags="$1"
+  local container_name
+  local distro
+  local img
+  local img_layers
+  local tmp_img
+  local tmp_img_layers
+  local tmp_output
```

Comment:
This variable looks very generic, and it is hard to guess what it does from its name. Fwiu, you want to get part of the layer image name, right? If so, rename this variable to something that expresses that idea.